src/client/failure/event-adapter.test.ts

```diff
@@ -1,11 +1,18 @@
 import { MapDocumentNode, ProfileDocumentNode } from '@superfaceai/ast';
+import {
+  ProfileConfiguration,
+  Provider,
+  ProviderConfiguration,
+  UseCase,
+} from '@superfaceai/one-sdk';
+import { SuperfaceClientBase } from '@superfaceai/one-sdk/dist/client/public/client';
```

Comment:
these are not the imports you're looking for :grin:

Reply:
Yep, this file is a mess. I am rewriting it now 🙂